src/utils/snaps.ts

```diff
@@ -141,3 +141,22 @@ export function getLatestSnapVersion(snap: VerifiedSnap) {
 
   return latest;
 }
+
+/**
+ * Get checksum of the latest version of the given Snap.
+ *
+ * @param snap - The Snap to get the latest version for.
+ * @param latestVersion - The latest version of the Snap.
+ * @returns The checksum value of latest version of the Snap.
+ */
+export function getLatestSnapVersionChecksum(
+  snap: VerifiedSnap,
+  latestVersion: string,
+) {
+  const { versions } = snap;
+
+  if (!versions[latestVersion]) {
+    return '';
```

Comment:
if it is empty, it is a unexpected error case, which we never side a message with empty id, so either handle in here or handle in generate node

Reply:
have change to throw exception here. 

So what will happen is if our registry data source provide a verified snap if doesnt have latest version data, our app will immediately throw error on generate node. anyway the error message will indicate which version data is missing in the registry.